src/query-follower/query-follower.ts

```diff
@@ -29,13 +29,39 @@ let J = JSON.stringify;
 
 //================================================================================
 
+/**
```

Comment:
The comments are duplicated in the types file and the implementation file?

Reply:
Yes. I've done this for QueryFollower and StorageAsync because there will be no hints for end users that all the docs for storage are actually on this interface. On the web, command line, or IDE they will simply see an undocumented class. My hope is that Deno will do this for us in the future ([see here](https://github.com/denoland/deno_doc/issues/136)), and that duplicating comments across two classes is an acceptable cost for the time being.

Reply:
I think it's dangerous to have duplicated documentation lying around, I'm pretty sure we'll update one and not the other at some point.  But I get that it's important for the docs.

I would say "let's have the class docs link back to the type docs" but then it still won't show up in editor hints.

We'll just have to be careful about the duplicated docs for the time being, I guess.